BrainPortal/lib/scir_lsf.rb

```diff
@@ -28,73 +28,62 @@ class ScirLsf < Scir
 
   class Session < Scir::Session #:nodoc:
 
-    def update_job_info_cache #:nodoc:
-      # FIXME: modify this method based
+    def update_job_info_cache #:nodoc:      
       # on the implementation of bstat.  This method is supposed to
       # list all the statuses of the running tasks, and to keep it in
       # the @job_info_cache array.
-      out, err = bash_this_and_capture_out_err("bstat -f -u #{CBRAIN::Rails_UserName.to_s.bash_escape}")
-      raise "Cannot get output of 'bstat -f' ?!?" if out.blank? && ! err.blank?
+      out, err = bash_this_and_capture_out_err("bjobs -a -noheader -J #{shell_escape(self.name)}")
```

Comment:
-J should be removed. The following option should be added: -u #{CBRAIN::Rails_UserName.to_s.bash_escape}